### DISCUSSION TITLE
next-intl instead of i18next?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [refinedev/refine#6054] [General]
Hi guys, anyone knows how to integrate **next-intl** instead of **i18next**?

**Comment by ghost** [comment]
hi guys, anyone can help?

**Comment by BatuhanW** [comment]
Hey @vitalijalbu @vitali-bc you can do the integration as seen in the next-intl docs. Also you can read our relevant guide: https://refine.dev/docs/guides-concepts/i18n/
